README.md

```diff
@@ -17,20 +17,20 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined **(classified as 'special' data type though not a primary)
 [] NaN
-[] Integers
-[] Arrays
-[] Null
+[X] Integers
+[No...but] Arrays **(called a 'composite' data type but is not a primary data type. It contains a list of primary data types (Strings, Booleans, Numbers/Integers))(overthinking it? Yeah... dern it.)
+[X] Null
 ```
```

Comment:
:+1: Technically "Number" is a Javascript data type, not "Integers." But not counting it as wrong.